src/RaisedButton/RaisedButton.spec.js

```diff
@@ -91,4 +91,16 @@ describe('<RaisedButton />', () => {
       );
     });
   });
+
+  it('inherits fontSize from theme', () => {
+    const wrapper = mountWithContext(
```

Comment:
Reading at the source code. I say it does.
Event, if we were using an internal `RaisedButtonLabel`, we could just make sure the right style props is applied.